src/packagedcode/rubygems.py

```diff
@@ -314,6 +320,13 @@ def build_rubygem_package(gem_data, download_url=None, package_url=None):
         declared_license=declared_license,
     )
 
+    # Since the gem spec doc is not clear https://guides.rubygems.org
+    # /specification-reference/#licenseo, we will treat a list of licenses and a
+    # conjunction for now (e.g. AND)
+    license = gem_data.get('license')
+    licenses = gem_data.get('licenses')
+    package = licenses_mapper(license, licenses, package)
```

Comment:
do not pass the package around.... instead get a declared_license list back and add it to the package creation above.... so this can and should be moved before the package creation

Reply:
Fixed.